### PR TITLE
Make blockframes prettier

### PR DESCRIPTION
**Pull request by ioaNikas**

### CODE REVIEW

File: libs/material/src/lib/delivery/+state/delivery.service.ts [hunk 1]
```diff
@@ -175,7 +177,9 @@ export class DeliveryService {
         ? deliveryStakeholder.authorizations
         : [...deliveryStakeholder.authorizations, authorization];
       this.db
-        .doc<Stakeholder>(`deliveries/${this.query.getActiveId()}/stakeholders/${deliveryStakeholder.id}`)
+        .doc<Stakeholder>(
+          `deliveries/${this.query.getActiveId()}/stakeholders/${deliveryStakeholder.id}`
+        )
```
Comment (Hugo-Hontans) [inline libs/material/src/lib/delivery/+state/delivery.service.ts]: indent correctly

File: libs/material/src/lib/delivery/delivery-list/delivery-list.component.ts [hunk 2]
```diff
@@ -18,12 +18,11 @@ import { takeWhile } from 'rxjs/operators';
 })
 export class DeliveryListComponent implements OnInit, OnDestroy {
   public movie$: Observable<Movie>;
-  public deliveries$: Observable<Delivery[]>;
+  public deliveries$: any;
```
Comment (Hugo-Hontans) [inline libs/material/src/lib/delivery/delivery-list/delivery-list.component.ts]: type observable

File: libs/material/src/lib/delivery/+state/delivery.query.ts [hunk 3]
```diff
@@ -30,6 +31,7 @@ export class DeliveryQuery extends QueryEntity<DeliveryState, Delivery> {
     private stakeholderQuery: StakeholderQuery,
     private organizationQuery: OrganizationQuery,
     private db: AngularFirestore,
+    private fireQuery: FireQuery
```
Comment (GrandSchtroumpf) [inline libs/material/src/lib/delivery/+state/delivery.query.ts]: Don't put it inside the Query.

File: libs/material/src/lib/delivery/delivery-list/delivery-list.component.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 <main class="container">
   <section *ngIf="(movie$ | async) as movie" class="delivery-list">
-    <article fxLayoutAlign="space-between">
+    <article class="sticky" fxLayoutAlign="space-between">
```
Comment (GrandSchtroumpf) [inline libs/material/src/lib/delivery/delivery-list/delivery-list.component.html]: Sticky class is commented. You don't use it

**Pull request closed without merging**